services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -515,7 +515,7 @@ private void processSlot() {
       this.eventBus.post(new BroadcastAggregatesEvent(nodeSlot.getValue()));
       nodeSlot.inc();
     } catch (InterruptedException e) {
-      LOG.error("onTick: {}", e.toString(), e);
```

Comment:
its super common to get exceptions here, and they're never anything other than InterruptedException, and the root cause doesn't seem to be set ,so i've reduced this to debug level.